Error in Cmake

/opt/libheif/libheif/plugins/heif_encoder_aom.cc: In function ‘heif_error aom_encode_image(void*, const heif_image*, heif_image_input_class)’:
/opt/libheif/libheif/plugins/heif_encoder_aom.cc:846:27: error: ‘AOM_USAGE_GOOD_QUALITY’ was not declared in this scope
  846 |   unsigned int aomUsage = AOM_USAGE_GOOD_QUALITY;
      |                           ^~~~~~~~~~~~~~~~~~~~~~
/opt/libheif/libheif/plugins/heif_encoder_aom.cc:849:16: error: ‘AOM_USAGE_REALTIME’ was not declared in this scope
  849 |     aomUsage = AOM_USAGE_REALTIME;
      |                ^~~~~~~~~~~~~~~~~~
make[2]: *** [libheif/CMakeFiles/heif.dir/build.make:284: libheif/CMakeFiles/heif.dir/plugins/heif_encoder_aom.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:320: libheif/CMakeFiles/heif.dir/all] Error 2
make: *** [Makefile:152: all] Error

it happened when i make code, does anyone know what this is about？

--- a/cmake/modules/FindAOM.cmake
+++ b/cmake/modules/FindAOM.cmake
@@ -1,4 +1,5 @@
 include(LibFindMacros)
+include(CheckSymbolExists)
 
 libfind_pkg_check_modules(AOM_PKGCONF aom)
 
@@ -7,6 +8,8 @@ find_path(AOM_INCLUDE_DIR
     HINTS ${AOM_PKGCONF_INCLUDE_DIRS} ${AOM_PKGCONF_INCLUDEDIR}
     PATH_SUFFIXES AOM
 )
+
+check_symbol_exists(AOM_USAGE_GOOD_QUALITY ${AOM_INCLUDE_DIR}/aom/aom_encoder.h aom_usage_flag_exists)
 
 find_library(AOM_LIBRARY
     NAMES libaom aom
@@ -19,7 +22,7 @@ else()
     set(AOM_DECODER_FOUND NO)
 endif()
 
-if(EXISTS "${AOM_INCLUDE_DIR}/aom/aom_encoder.h")
+if((EXISTS "${AOM_INCLUDE_DIR}/aom/aom_encoder.h") AND (DEFINED aom_usage_flag_exists))
     set(AOM_ENCODER_FOUND YES)
 else()
     set(AOM_ENCODER_FOUND NO)

